src/drivers/Bma421.cpp

```diff
@@ -126,7 +109,4 @@ void Bma421::SoftReset() {
     isResetOk = true;
     nrf_delay_ms(1);
   }
-}
-Bma421::DeviceTypes Bma421::DeviceType() const {
-  return deviceType;
-}
+}
```

Comment:
please add newline at the end of the file 

```suggestion
}

```